diff --git a/src/main/java/com/okbo_projects/domain/board/controller/BoardController.java b/src/main/java/com/okbo_projects/domain/board/controller/BoardController.java
index 5f0b109..34bd213 100644
--- a/src/main/java/com/okbo_projects/domain/board/controller/BoardController.java
+++ b/src/main/java/com/okbo_projects/domain/board/controller/BoardController.java
@@ -1,11 +1,19 @@
 package com.okbo_projects.domain.board.controller;


+import com.okbo_projects.domain.board.model.request.CreateBoardRequest;
+import com.okbo_projects.domain.board.model.request.UpdateBoardRequest;
+import com.okbo_projects.domain.board.model.response.CreateBoardResponse;
+import com.okbo_projects.domain.board.model.response.DetailedInquiryBoardResponse;
+import com.okbo_projects.domain.board.model.response.UpdateBoardResponse;
+import com.okbo_projects.domain.board.model.response.ViewListOfMyArticlesWrittenResponse;
 import com.okbo_projects.domain.board.service.BoardService;
 import lombok.RequiredArgsConstructor;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.security.Principal;
+import java.util.List;
 
 @RestController
 @RequestMapping("/boards")
@@ -15,4 +23,30 @@ public class BoardController {
     private final BoardService boardService;
 
 
+    //게시글 생성
+    @PostMapping("/create")
+    public ResponseEntity<CreateBoardResponse> createBoard(@RequestBody CreateBoardRequest request, Principal principal){
+
+        return ResponseEntity.ok(boardService.createBoard(1L, request));
+
+    }
+
+    //게시글 수정
+    @PutMapping("/{id}")
+    public ResponseEntity<UpdateBoardResponse> updateBoard(@PathVariable Long id, @RequestBody UpdateBoardRequest request){
+        return ResponseEntity.ok(boardService.updateBoard(id, request));
+    }
+    //게시글 상세조회
+    @GetMapping("/{id}")
+    public ResponseEntity<DetailedInquiryBoardResponse> detailedInquiryBoard(@PathVariable Long id){
+        return ResponseEntity.ok(DetailedInquiryBoardResponse.from(boardService.detailedInquiryBoard(id)));
+
+    }
+    //내가 작성한 게시글 목록 조회
+    @GetMapping("/myboard")
+    public ResponseEntity<List<ViewListOfMyArticlesWrittenResponse>> viewListOfMyArticlesWritten(Principal principal){
+        return ResponseEntity.ok(boardService.viewListOfMyArticlesWritten(1L));
+    }
+
+
 }
diff --git a/src/main/java/com/okbo_projects/domain/board/model/dto/BoardDto.java b/src/main/java/com/okbo_projects/domain/board/model/dto/BoardDto.java
index 5a2f5c7..45ff237 100644
--- a/src/main/java/com/okbo_projects/domain/board/model/dto/BoardDto.java
+++ b/src/main/java/com/okbo_projects/domain/board/model/dto/BoardDto.java
@@ -24,15 +24,15 @@ public class BoardDto {
     private LocalDateTime modifiedAt;
 
     //기능
-//    public static BoardDto from(Board board) {
-//        return new BoardDto(
-//                board.getId(),
-//                board.getTitle(),
-//                board.getTeam(),
-//                board.getContent(),
-//                UserDto.from(board.getWriter()),
-//                board.getCreatedAt(),
-//                board.getModifiedAt()
-//        );
-//    }
+    public static BoardDto from(Board board) {
+        return new BoardDto(
+                board.getId(),
+                board.getTitle(),
+                board.getTeam(),
+                board.getContent(),
+                UserDto.from(board.getWriter()),
+                board.getCreatedAt(),
+                board.getModifiedAt()
+        );
+    }
 }
diff --git a/src/main/java/com/okbo_projects/domain/board/model/request/CreateBoardResponse.java b/src/main/java/com/okbo_projects/domain/board/model/request/CreateBoardResponse.java
index b85f736..ac5c9e8 100644
--- a/src/main/java/com/okbo_projects/domain/board/model/request/CreateBoardResponse.java
+++ b/src/main/java/com/okbo_projects/domain/board/model/request/CreateBoardResponse.java
@@ -5,5 +5,5 @@ import lombok.NoArgsConstructor;
 
 @Getter
 @NoArgsConstructor
-public class BoardCreatRequest {
+public class CreateBoardResponse {
 }
diff --git a/src/main/java/com/okbo_projects/domain/board/model/response/CreatBoardResponse.java b/src/main/java/com/okbo_projects/domain/board/model/response/CreatBoardResponse.java
index 2060d93..53b359f 100644
--- a/src/main/java/com/okbo_projects/domain/board/model/response/CreatBoardResponse.java
+++ b/src/main/java/com/okbo_projects/domain/board/model/response/CreatBoardResponse.java
@@ -5,5 +5,5 @@ import lombok.NoArgsConstructor;
 
 @Getter
 @NoArgsConstructor
-public class BoardCreatResponse {
+public class CreatBoardResponse {
 }
